idd/Energy+.idd.in

```diff
@@ -108149,6 +108149,152 @@ Meter:CustomDecrement,
+       \unique-object
+  A1 , \field Output CSV
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A2 , \field Output MTR
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A3 , \field Output ESO
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A4 , \field Output EIO
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A5 , \field Output AUDIT
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A6 , \field Output Zone Sizing
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A7 , \field Output System Sizing
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A8 , \field Output DXF
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A9 , \field Output BND
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A10, \field Output RDD
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A11, \field Output MDD
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A12, \field Output MTD
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A13, \field Output END
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A14, \field Output SHD
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A15, \field Output DFS
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A16, \field Output GLHE
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A17, \field Output DelightIn
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A18, \field Output DelightELdmp
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A19, \field Output DelightDFdmp
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A20, \field Output EDD
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A21, \field Output DBG
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A22, \field Output PerfLog
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A23, \field Output SLN
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A24, \field Output SCI
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A25, \field Output WRL
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A26, \field Output Screen
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A27, \field Output ExtShd
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+  A28; \field Output Tarcog
+       \note Not Implemented Yet
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+
```

Comment:
More file types to include: Table, Daylighting Map, VRML (or is that what WRL is above?), SQL, XML, JSON? I'm assuming the goal for this object is to provide one place to enable/disable every possible output file. Maybe that's too ambitious.

Reply:
I could definitely add all those files, however, my initial goal was to not add them since they are controlled by other objects. Meaning, if those objects are not present then those outputs are already not output by the simulation. I considered one master enable/disable control object, but I thought that could also lead to user confusion with multiple objects controlling output. However, I am open to suggestions and thoughts.

Reply:
My thought was that this object could be a way to turn off files, even if they require other objects present to create them. Then one could preserve those settings in the idf but disable the files as desired. If this object is limited to only files that don't have some other object to control/create them, then the list shrinks to eso, audit, zone sizing, system sizing, bnd, mtd, end, and maybe a few others. For example, rdd, mdd, dxf, all have objects that trigger them. 

I'm guessing this object started as a way to activate writing the csv file directly, because there wasn't any place to turn that on/off. So, you could shrink this to OutputControl:TimeSeries, csv yes/no, eso yes/no. But then that overlaps with output:sqlite, and output:json.